src/component/utils/Modal.tsx

```diff
@@ -35,44 +42,39 @@ const Modal = ({ isOpen, onCloseModal, children, size }: ModalProps) => {
     return candidate.includes(size) ? size : "basic";
   };
 
-  const onClickBackdrop = e => {
+  const closeModal = (e: MouseEvent<HTMLElement>) => {
     e.preventDefault();
     onCloseModal();
   };
 
   useEffect(() => {}, []);
 
-  const onClickContainer = e => {
-    // 이벤트 버블링 방지
+  const preventEventBubbling = (e: MouseEvent<HTMLElement>) => {
     e.stopPropagation();
   };
-  // 선언된 위치가 아닌 지정된 위치에 생성
-  const modalPosition = document.getElementById("modal");
 
-  return createPortal(
-    <>
-      <div
-        className="modal__backdrop"
-        onClick={onClickBackdrop}
-        aria-hidden="true"
-      >
+  const modalPosition = document.getElementById("portal");
+
+  return (
+    modalPosition &&
+    createPortal(
+      <>
         <div
-          className={`modal__container ${modalSize()}`}
-          onClick={onClickContainer}
-          onKeyPress={onClickContainer}
+          className="modal__backdrop"
+          onClick={closeModal}
+          aria-hidden="true"
```

Comment:
```suggestion
          role="dialog"
          aria-modal="true"
```
모달용 aria 태그가 있다고 합니다. 사용해보면 좋을 것 같습니다.
참고 링크:
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal

Reply:
해당 코드는 backdrop이 큰 의미없이 배경으로 깔리는 검은 화면이라 접근성에서 고려하지 않기 위해 aria-hidden 옵션을 준 코드입니다
실제 모달로 작동하는 부분은 다른 곳입니다
role="dialog" 옵션 등은 그 아래 67번 div 쪽으로 추가하겠습니다!!


다만 언급하신 레퍼런스에도 기재되어 있지만 modal 접근성은 단순히 role과 aria 옵션만으로 해결되는게 아니라 더 개선이 필요할 것 같습니다. 
특히 포커싱 관리가 제대로 되지 않아서 접근성을 고려한 모달로는 부족한 점이 아직 남아있네요
앞으로 차근차근 추가해보시죠!!!